FAQ.md

```diff
@@ -85,11 +85,14 @@ Here are some suggestions for when you could use `dep` or `go get`:
 -[@sdboyer in #376](https://github.com/golang/dep/issues/376#issuecomment-294045873)
 
 ## Why is `dep` ignoring a version constraint in the manifest?
-Only your project's directly imported dependencies are affected by a `dependencies` entry
+Only your project's directly imported dependencies are affected by a `constraint` entry
 in the manifest. Transitive dependencies are unaffected.
 
 Use an `overrides` entry for transitive dependencies.
 
+To pin a version of direct dependency in manifest, prefix the version with `=`.
+e.g. `version = "=x.y.z"`
```

Comment:
Let's show a full example with the `[constraint]` header so that it's clear which one to use.